scripts/loadavg.php

```diff
@@ -0,0 +1,7 @@
+<?php
+echo json_encode(@sys_getloadavg());
+<?php
+// Put this in your MySQL Server if it's external
```

Comment:
Maybe document this in the slave setup example too in the Wiki.

Reply:
Will do

Reply:
Added this to the load checking instead, seems to make more sense in a way
The Slave setup Wiki probably needs to be reviewed as per my previous pull request containing master/slave without extensions, what do you think?